formula/src/main/java/com/instacart/formula/Formula.kt

```diff
@@ -44,4 +44,8 @@ interface Formula<Input, State, RenderModel> {
         state: State,
         context: FormulaContext<State>
     ): Evaluation<RenderModel>
+
+    override fun implementation(): Formula<Input, *, RenderModel> {
```

Comment:
Will this inherit docs? Otherwise, we might want to add docs on why/when you would override this.

Reply:
Looks like it.